_includes/header.html

```diff
@@ -28,19 +27,16 @@
 			<nav>
 				<div class="nav-bar">
 					<div class="module left">
-						<a href="https://fossasia.org/">
+						<a href="./#welcome">
 							<img class="logo logo-dark" alt="FOSSASIA" src="img/fossasia-dark.png" width="110px" height="18.69">
 						</a>
 					</div>
 					<div class="module widget-handle mobile-toggle right visible-sm visible-xs">
-						<i class="ti-menu"></i>
+						<i class="fa fa-bar"></i>
```

Comment:
Why is this diff here? Err.

Reply:
That is because he removed `themify-icons.css`. And this was the only line using themify icons, so he changed it into FontAwesome.

Reply:
That is because he removed `themify-icons.css`. And this was the only line using themify icons, so he changed it into FontAwesome.

Reply:
@duttaditya18 https://github.com/fossasia/gci17.fossasia.org/pull/557

Reply:
Themify was removed earlier than this. Precise ref: https://github.com/fossasia/gci17.fossasia.org/pull/557/files#diff-822e9cf54ee310f9a7ebd0d16c9ae066R35

Reply:
That is true. Then what exactly did he delete? I don't get it.

Reply:
@duttaditya18 my bad, seems @nikit19 did a direct revert. But I did'nt get the reason for it. Was it a mistake?

Reply:
@abishekvashok I didn't realize that. Yes it got reverted by mistake

Reply:
Same

Reply:
@nikit19 What changes got reverted? Also, Please think twice before reverting 😄 
If making a revert than please open issue for the same so that other devs can get an update. Thanks

Reply:
@vaibhavsingh97 #557  was reverted. we can't afford a force push.... can we?

Reply:
@vaibhavsingh97 I am really sorry about the inconvenience that has been caused.
[#585](https://github.com/fossasia/gci17.fossasia.org/pull/585) this was to revert some other commit but by mistake [#557](https://github.com/fossasia/gci17.fossasia.org/pull/557) got reverted

Reply:
@abishekvashok if I am right your changes are still there in the main repository ? We are seeing the old changes in this PR
If you look in the main repo it is `<i class="fa fa-bar"></i>`
see here
https://github.com/fossasia/gci17.fossasia.org/blob/gh-pages/_includes/header.html